src/components/DetailsMenu/DetailsMenuCard/ButtonRow/index.tsx

```diff
@@ -21,7 +21,7 @@ const ButtonRow: React.FC = () => {
   const { htmlLogURL, jobLogsURL, legacyJobLogsURL, lobsterURL, rawLogURL } =
     logMetadata || {};
   const tooltipText = bookmarks.length
-    ? "Copy Bookmarked Lines In Jira Format"
+    ? "Copy Bookmarked Lines In JIRA Format"
```

Comment:
Sentence case is more in line with the other tooltips
```suggestion
    ? "Copy bookmarked lines in JIRA format"
```